twitterscraper/main.py

```diff
@@ -113,13 +113,35 @@ def main():
                 with open(args.output, "w", encoding="utf-8") as output:
                     if args.csv:
                         f = csv.writer(output, delimiter=";")
-                        f.writerow(["username", "fullname","user_id", "tweet_id", "tweet_url", "timestamp","timestamp_epochs",
-                                    "replies", "retweets", "likes", "is_retweet", "retweeter_username" , "retweeter_userid" ,
-                                    "retweet_id","text", "html"])
+                        f.writerow([
+                            # user name & id
+                            "screen_name", "username", "user_id",
+                            # tweet basic data
+                            "tweet_id", "tweet_url", "timestamp", "timestamp_epochs",
+                            # tweet text
+                            "text", "text_html", "links", "hashtags",
+                            # tweet media
+                            "has_media", "img_urls", "video_url",
+                            # tweet actions numbers
+                            "likes", "retweets", "replies", "is_replied",
+                            # detail of reply to others
+                            "is_reply_to", "parent_tweet_id", "reply_to_users"
```

Comment:
While I can understand the value of grouping the column names I personally would not include the comments. They add visual noise to the code and in my opinion don't provide much value.